File: CS/WindowsApplication3/Main.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;
using DevExpress.XtraEditors;
using DevExpress.XtraBars.Ribbon;
using DevExpress.XtraBars.Ribbon.ViewInfo;
using DevExpress.XtraBars;


namespace DXSample {
    public partial class Main: XtraForm {

        RibbonDragDropHelper helper;

        public Main() {
            InitializeComponent();
        }
       
        private void OnFormLoad(object sender, EventArgs e) {
            foreach(BarItem item in ribbonControl1.Items)
                ribbonControl1.Toolbar.ItemLinks.Add(item);
            helper = new RibbonDragDropHelper(ribbonControl1);
            helper.EnableQATDragDrop();
        }

        protected override void OnFormClosing(FormClosingEventArgs e)
        {
            base.OnFormClosing(e);
            helper.DisableQATDragDrop();
        }
    }
}

File: VB/WindowsApplication3/Main.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms
Imports DevExpress.XtraEditors
Imports DevExpress.XtraBars.Ribbon
Imports DevExpress.XtraBars.Ribbon.ViewInfo
Imports DevExpress.XtraBars


Namespace DXSample
	Partial Public Class Main
		Inherits XtraForm

		Private helper As RibbonDragDropHelper

		Public Sub New()
			InitializeComponent()
		End Sub

		Private Sub OnFormLoad(ByVal sender As Object, ByVal e As EventArgs) Handles MyBase.Load
			For Each item As BarItem In ribbonControl1.Items
				ribbonControl1.Toolbar.ItemLinks.Add(item)
			Next item
			helper = New RibbonDragDropHelper(ribbonControl1)
			helper.EnableQATDragDrop()
		End Sub

		Protected Overrides Sub OnFormClosing(ByVal e As FormClosingEventArgs)
			MyBase.OnFormClosing(e)
			helper.DisableQATDragDrop()
		End Sub
	End Class
End Namespace
